File: 1-two-sum/1-two-sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        for(int i=0;i<nums.Length;i++){
            for(int j=i+1;j<nums.Length;j++){
                if(nums[i]+nums[j]==target)
                {
                    int[] soltn=new int[2];
                    soltn[0]=i;
                    soltn[1]=j;
                    return soltn;
                }
            }
        }
        return new int[2];
    }
}

File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        sum_pairs = dict()
        
        for idx in range(len(nums)):
            if sum_pairs.get(nums[idx]) is not None:
                return [sum_pairs.get(nums[idx]), idx]
            else:
                sum_pairs[target - nums[idx]] = idx
        
        return list()